client/src/pages/Resource/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
+		resourceService.getOne(id).then(setResource);
```

Comment:
If resourceService.getOne(id) fails, there is no error-handling mechanism here.

Suggestion: Maybe adding a catch block helps to handle any issues with data fetching.